services/libs/httpclients/src/searchSyncApiClient.ts

```diff
@@ -0,0 +1,160 @@
+import axios from 'axios'
+
+export class SearchSyncApiClient {
+  private searchSyncApi
+
+  constructor() {
+    this.searchSyncApi = axios.create({
+      baseURL: process.env.SEARCH_SYNC_API_URL,
```

Comment:
can you make it so we can pass this through constructor when we initialize SearchSyncApiClient?